utils/cookies_login.py

```
@@ -0,0 +1,348 @@
+}
+
+class LoginStatus:
+    SUCCESS = 0
+    FAILED = -1
+    TIMEOUT = -2
+
+# ==========================================
+# 🛡️ 安全模块：脱敏、校验与文件锁
+# ==========================================
+def mask_string(s: str) -> str:
+    """对敏感凭证进行打码处理，防止屏幕偷窥或日志泄露"""
```

Ruff (0.15.1)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)